### PR TITLE
Windows cleanup

### PR DESCRIPTION
**Pull request by Benjamintf1**
# Description

Remove carriage returns from end of line for windows
ensure blackbox doesn't crash during tls errors

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing performed?

- [x] Unit tests
- [x] Integration tests
- [x] Acceptance tests

## Checklist:

- [x] This PR is being made against the `main` branch, or relevant version branch
- [ ] I have made corresponding changes to the documentation
- [x] I have added testing for my changes

If you have any questions, or want to get attention for a PR or issue please reach out on the [#logging-and-metrics channel in the cloudfoundry slack](https://cloudfoundry.slack.com/archives/CUW93AF3M)

### CODE REVIEW

File: integration/syslog_test.go [hunk 1]
```diff
@@ -751,13 +704,68 @@ var _ = Describe("Blackbox", func() {
...
+			err := tlsserver.Run()
+			Expect(err).ToNot(HaveOccurred())
+			blackboxRunner = NewBlackboxRunner(blackboxPath)
+		})
+
+		AfterEach(func() {
+			tlsserver.Stop()
+		})
+
+		It("doesn't cause blackbox to fail", func() {
```
Comment (ctlong) [inline integration/syslog_test.go:738]: This test is just to future-proof blackbox against being changed to crash on bad TLS errors, is that it?

File: integration/tls_syslog_server.go [hunk 2]
```diff
@@ -35,8 +36,12 @@ func (s *TLSSyslogServer) Run() error {
 		return errors.New("failed to apped CA")
 	}
 
+	certPrefix := "fixtures/server"
+	if s.CertPrefixOverride != "" {
+		certPrefix = s.CertPrefixOverride
+	}
 	// Listen for incoming connections.
```
Comment (ctlong) [inline integration/tls_syslog_server.go:43]: Move comment above the `if` statement for style.

File: integration/fixtures/regenerate-certs.sh [hunk 3]
```diff
@@ -10,6 +10,9 @@ rm -rf out
 certstrap init --common-name "ca" --passphrase ""
 certstrap request-cert --common-name "server" --domain "localhost" --ip "127.0.0.1" --passphrase ""
 certstrap sign server --CA "ca"
+certstrap init --common-name "ca-bad" --passphrase ""
+certstrap request-cert --common-name "server-bad" --passphrase ""
+certstrap sign server-bad --CA "ca-bad"
```
Comment (ctlong) [inline integration/fixtures/regenerate-certs.sh:15]: Why save `ca-bad.crl` and `ca-bad.crt` when we don't actually use them in tests?

**Review by ctlong — CHANGES_REQUESTED** [review]
Hey @Benjamintf1, @duanemay and I have left some review comments.

File: integration/syslog_test.go [hunk 4]
```diff
@@ -318,8 +299,7 @@ var _ = Describe("Blackbox", func() {
 			config.StructuredDataMap = map[string]string{"test": "1"}
 			blackboxRunner.StartWithConfig(config, 1)
 
-			logFile.WriteString("hello\n")
-			logFile.Sync()
+			Write(logFile, "hello\n", true, true)
```
[the remark this answers is not in the document]
Comment (Benjamintf1) [reply integration/syslog_test.go:302]: It's not being used later so it shouldn't matter? I can change back, or try and do it more completely tho.

File: integration/syslog_test.go [hunk 5]
```diff
@@ -331,14 +311,12 @@ var _ = Describe("Blackbox", func() {
...
-			logFile.WriteString("already present\n")
-			logFile.Sync()
+			Write(logFile, "already present\n", true, false)
 
 			config := buildConfig(logDir)
 			blackboxRunner.StartWithConfig(config, 1)
 
-			logFile.WriteString("hello\n")
-			logFile.Sync()
+			Write(logFile, "hello\n", true, true)
```
[the remark this answers is not in the document]
Comment (Benjamintf1) [reply integration/syslog_test.go:319]: It's not being used later so it shouldn't matter? I can change back, or try and do it more completely tho.

File: integration/syslog_test.go [hunk 6]
```diff
@@ -751,13 +704,68 @@ var _ = Describe("Blackbox", func() {
...
 	})
+
+	Context("When the server uses bad tls", func() {
+		var address string
+		var buffer *gbytes.Buffer
+		var tlsserver TLSSyslogServer
+		var blackboxRunner *BlackboxRunner
+
+		BeforeEach(func() {
+			address = fmt.Sprintf("127.0.0.1:%d", 9090+GinkgoParallelNode())
```
[the remark this answers is not in the document]
Comment (Benjamintf1) [reply integration/syslog_test.go:722]: Maybe can look into this, it was just copied from another test.

File: integration/syslog_test.go (hunk 1 above)
Comment (Benjamintf1) [reply integration/syslog_test.go:738]: yup

File: integration/fixtures/regenerate-certs.sh (hunk 3 above)
Comment (Benjamintf1) [reply integration/fixtures/regenerate-certs.sh:15]: Fair. I needed them i think to be on a different ca, but I'll check again.

File: integration/tls_syslog_server.go (hunk 2 above)
Comment (Benjamintf1) [reply integration/tls_syslog_server.go:43]: Actually I moved it below cause it seems to refer to a different line entirely

File: integration/fixtures/regenerate-certs.sh (hunk 3 above)
Comment (Benjamintf1) [reply integration/fixtures/regenerate-certs.sh:15]: didn't need a different ca. idk how i was setting up the test wrong before.

**Review by ctlong — APPROVED** [review]
LGTM

**Pull request merged**